README.md

```diff
@@ -1,2 +1,70 @@
-# enhancements
-Enhancement Tracking and Backlog Repo for Istio Releases
+# Enhancement Tracking and Backlog
+Enhancement Tracking and Backlog Repo for Istio Releases. Owned by TOC.
+
+This repo contains issues and Istio enhancement proposal. These issues are umbrellas for new enhancements to be added to Istio. An enhancement usually takes multiple releases to complete. And an enhancement can be tracked as backlog items before work begins. An enhancement may be filed once there is consensus in at least one Istio working group.
```

Comment:
When should I file an enhancement here and when should I file one as a "type/enhancement" under istio/istio?

Reply:
the idea is type/enhancement should be recorded here instead

Reply:
Concrete example: https://github.com/istio/istio/issues/19552

Would this be an IEP? I would not think so?

Reply:
Agree re tech writer, but lets get the content nailed down first then we can improve it